pkg/systemd/generate/containers.go

```diff
@@ -68,7 +68,7 @@ type containerInfo struct {
 
 	// If not nil, the container is part of the pod.  We can use the
 	// podInfo to extract the relevant data.
-	pod *podInfo
+	Pod *podInfo
```

Comment:
I don't think exporting is necessary

Reply:
It is a template can only access exported functions

Reply:
Thanks for clarifying!